### ISSUE TITLE
magic-numbers-deprecation

### ISSUE THREAD

**Issue by imark73** [peaceduck/remootio#7]
Hi

I noticed in my logs the following, and wonder whether a fix is planned before the deprecation period is finished -
"Core 2025.1. Instead it should use <CoverEntityFeature.OPEN|CLOSE: 3>, please report it to the author of the 'remootio' custom integration and reference https://developers.home-assistant.io/blog/2023/12/28/support-feature-magic-numbers-deprecation"
Thanks!

**Comment by TerminalAddict** [comment]
bumping this ... 
The only reference i can see is in custom_components/remootio/cover.py
```_attr_supported_features = cover.SUPPORT_OPEN | cover.SUPPORT_CLOSE ```

A copy of the logs justy in case 
```
SUPPORT_OPEN was used from remootio, this is a deprecated constant which will be removed in HA Core 2025.1. Use CoverEntityFeature.OPEN instead, please report it to the author of the 'remootio' custom integration
SUPPORT_CLOSE was used from remootio, this is a deprecated constant which will be removed in HA Core 2025.1. Use CoverEntityFeature.CLOSE instead, please report it to the author of the 'remootio' custom integration

Entity None (<class 'custom_components.remootio.cover.RemootioCover'>) is using deprecated supported features values which will be removed in HA Core 2025.1. Instead it should use <CoverEntityFeature.OPEN|CLOSE: 3>, please report it to the author of the 'remootio' custom integration and reference https://developers.home-assistant.io/blog/2023/12/28/support-feature-magic-numbers-deprecation
```

**Comment by peaceduck** [comment]
Thanks for reminding me! A lot of things to do before Christmas with work. BUT, this has to be fixed, of course. I will make sure to fix this! Can not promise before Christmas, but I should be able to get time to fix this at least before the New Year. I really need my garage door to work!
Sorry for not replying sooner.
/Fredrik

**Comment by my-axel** [comment]
No worries. December is always a busy time. Hope you'll have a few days off soon and can relax a little.

**Comment by imark73** [comment]
Thanks guys, much appreciated 🙂

**Comment by peaceduck** [comment]
So, I have now fixed this in release 2.0.2! It looks like it works fine for me. I will close this. Please open a new ticket if you have any problems with it!
